presto-docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -34,13 +34,64 @@ Presto restart. The refresh period is specified in the ``etc/access-control.prop
 
    security.refresh-period=1s
 
+Catalog, Schema, and Table Access
+---------------------------------
+
+Access to catalogs, schemas, tables, and views is controlled by the catalog, schema, and table
+rules.  The catalog rules are course grained rules used to restrict all access or write
+access to catalogs, but does not explicitly grant any specific schema or table permissions.
+The table and schema rules are used to specify who can can create, drop, alter, select, insert,
```

Comment:
maybe new paragraph

Reply:
I like this as an introductory summary paragraph that describe what the rules do at a high level so having these here make sense to me.

Reply:
Yes .. just add an empty line to break up this large paragraph into two.. but still leave it here..